dbsnap_verify/rds_funcs.py

```diff
@@ -0,0 +1,172 @@
+from operator import itemgetter
+
+from random import choice
+
+from string import (
+    letters,
+    digits,
+)
+
+VALID_SNAPSHOT_TYPES = ["automated", "manual"]
+
+
+def generate_password(size = 9, pool = None):
+    """
+    Return a system generated password.
+    :param size:
+        The desired length of the password to generate. (Default 9)
+    :param pool:
+        Pool of chars to choose from. (Default digits and letters [upper/lower])
+    :returns: String (raw password)
+    """
+    if pool == None:
+        pool = letters + digits
+    return ''.join( [ choice( pool ) for i in range( size ) ] )
+
+
+def get_available_snapshots(session, db_id, snapshot_type=None):
+    """Returns DB snapshots in the available state for a given db id.
+    Args:
```

Comment:
Looks like you're mixing python docstring comment styles? (see previous function).  I prefer the Google style (this one), so I'd go with that personally.

Reply:
fixed